ops/internal/deployer/state.go

```diff
@@ -260,3 +260,27 @@ func readL1ChainID(node *dasel.Node) (uint64, error) {
 	}
 	return uint64(l1ChainIDFloat), nil
 }
+
+func ReadL1ContractsRelease(node *dasel.Node) (string, error) {
+	l1ContractsReleaseNode, err := node.Query("appliedIntent.l1ContractsLocator")
```

Comment:
Ideally this is stable so we don't have to modify this function in future to keep up with OPD versions.